### PR TITLE
FAQ updates

### PR DESCRIPTION
**Pull request by jmklix**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: documents/FAQ.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+*__Jump To:__*
+* [Where should I start](#where-should-i-start)
+* [How do I enable logging](#how-do-i-enable-logging)
+* [I keep getting AWS_ERROR_MQTT_UNEXPECTED_HANGUP](#i-keep-getting-aws_error_mqtt_unexpected_hangup)
+* [I am experiencing deadlocks](#i-am-experiencing-deadlocks)
+* [Mac-Only TLS Behavior](#mac-only-tls-behavior)
+* [How do debug in VSCode?](#how-do-debug-in-vscode)
+* [What certificates do I need?](#what-certificates-do-i-need)
+* [I would like to build a browser application and got error "Property does not exist on type 'typeof import("\<path\>/node_modules/aws-crt/dist/**native**/*")](#browser-error)
+* [I still have more questions about the this sdk?](#i-still-have-more-questions-about-the-this-sdk)
```
Comment (sfod) [inline documents/FAQ.md:12]: I think "the" in the title is excess

File: README.md [hunk 2]
```diff
@@ -13,11 +13,11 @@ cross-platform, high-performance, secure, and reliable. The libraries are bound
 to JS by the [awscrt](https://github.com/awslabs/aws-crt-nodejs) package.
 
 *__Jump To:__*
-* [Installation](#Installation)
+* [Installation](#installation)
 * [Samples](samples)
-* [Getting Help](#Getting-Help)
+* [Getting Help](#getting-Help)
```
Comment (sfod) [inline README.md:18]: I think all letters in the anchor must be lowercase.
I couldn't find this rule in the github docs, but that's how github itself generates these anchors (the link to the left of each header). Also, other markdown docs state that anchors should be all lowercase.

**Review by sfod — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jmklix) [reply README.md]: Yes, thanks for pointing this out

File: documents/FAQ.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
 # Frequently Asked Questions
 
+*__Jump To:__*
+* [Where should I start](#where-should-i-start)
+* [How do I enable logging](#how-do-i-enable-logging)
+* [I keep getting AWS_ERROR_MQTT_UNEXPECTED_HANGUP](#i-keep-getting-aws_error_mqtt_unexpected_hangup)
+* [I am experiencing deadlocks](#i-am-experiencing-deadlocks)
```
Comment (bretambrose) [inline documents/FAQ.md:7]: deadlocks related to waiting in callbacks is not relevant to javascript

**Review by bretambrose — CHANGES_REQUESTED** [review]

File: documents/FAQ.md (hunk 3 above)
Comment (jmklix) [reply documents/FAQ.md]: thanks, removed that section

**Review by bretambrose — APPROVED** [review]

**Pull request merged**